Src/IronPythonTest/Conversions.cs

```diff
@@ -69,11 +70,11 @@ public class DictConversion {
         public static IList<object> ToIDictionary(IDictionary dict) {
             List<object> res = new List<object>();
             foreach (DictionaryEntry de in dict) {
-                res.Add(de.Key);
+                res.Add(PythonTuple.MakeTuple(de.Key));
```

Comment:
Not quite what I had in mind. I was thinking more like `PythonTuple.MakeTuple(de.Key, de.Value)`. I'm surprised the test is passing! I guess tuple comparison needs work (e.g. `(1,) < (None,)` should throw).

Reply:
I guess because we use comparison in `CompareArrays()`:
https://github.com/IronLanguages/ironpython3/blob/fa2d3972aaecc83283b205b47feb1a51883becc6/Src/IronPython/Runtime/Operations/PythonOps.cs#L688-L696

Reply:
Yeah, something for a separate PR I think.